DialogflowApp.ts

```diff
@@ -36,35 +42,40 @@ export class DialogflowApp extends App implements IPostMessageSent, IPostLivecha
         super(info, logger, accessors);
     }
 
-    public async onEnable(environment: IEnvironmentRead, configurationModify: IConfigurationModify): Promise<boolean> {
+    public async onEnable(): Promise<boolean> {
```

Comment:
`onEnabled` is defined as below. So we should not change the fn definition. Can you please add these args back, and may be adding `_` for the lint issue

```
/**
     * Method which is called when this App is enabled and can be called several
     * times during this instance's life time. Once after the `ititialize()` is called,
     * pending it doesn't throw an error, and then anytime the App is enabled by the user.
     * If this method, `onEnable()`, returns false, then this App will not
     * actually be enabled (ex: a setting isn't configured).
     *
     * @return whether the App should be enabled or not
     */
    onEnable(environment: IEnvironmentRead, configurationModify: IConfigurationModify): Promise<boolean>;
```